ci-operator/jobs/openshift/builder/openshift-builder-presubmits.yaml

```diff
@@ -63,3 +63,204 @@ presubmits:
             cpu: 10m
       serviceAccountName: ci-operator
     trigger: ((?m)^/test( all| images),?(\s+|$))
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/gcp
+    decorate: true
+    name: pull-ci-origin-e2e-gcp
+    rerun_command: /test gcp
+    skip_cloning: true
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --secret-dir=/usr/local/e2e-gcp-cluster-profile
+        - --template=/usr/local/e2e-gcp
+        - --target=e2e-gcp
+        - --target=[release:latest]
```

Comment:
I've never seen this before ... @smarterclayton what is this?

Reply:
this is the release payload target.  it's not necessary for e2e-gcp jobs, so you can remove it ben

Reply:
fyi i copied this from the origin config.